### PR TITLE
feat(sink): Support cassandra sink

### PR DESCRIPTION
**Pull request by xxhZs**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?
support cassandra sink. We can build cassandra sink like README

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Release note

If this PR includes changes that directly affect users or other significant modifications relevant to the community, kindly draft a release note to provide a concise summary of these changes. Please prioritize highlighting the impact these changes will have on users.

</details>

### CODE REVIEW

File: java/connector-node/risingwave-sink-cassandra/src/main/java/com/risingwave/connector/CassandraSink.java [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+                    break;
+                case UPDATE_INSERT:
+                    SinkRow old = updateRowCache.remove(0);
+                    if (old == null) {
+                        throw Status.FAILED_PRECONDITION
+                                .withDescription("UPDATE_INSERT without UPDATE_DELETE")
+                                .asRuntimeException();
+                    }
+                    batchBuilder.addStatement(
+                            bindUpdateInsertStatement(this.stmtMap.get("update"), old, row));
```
Comment (hzxa21) [inline java/connector-node/risingwave-sink-cassandra/src/main/java/com/risingwave/connector/CassandraSink.java:150]: FYI, cassandra is upsert by design. I think we can use insert statement instead of update statement here to simplify the codes. I am also okay of keeping the current implementation.

**Comment by lmatz** [comment]
As one user requested sink to SycllaDB and SycllaDB claims to be "compatible with Apache Cassandra", could you add the same test for SycllaDB?

SycllaDB also offers a docker image:https://hub.docker.com/r/scylladb/scylla/

**Comment by hzxa21** [comment]
> As one user requested sink to SycllaDB and SycllaDB claims to be "compatible with Apache Cassandra", could you add the same test for SycllaDB?
> 
> SycllaDB also offers a docker image:https://hub.docker.com/r/scylladb/scylla/

We have added demos for both Apache Cassandra and ScyllaDB. They both work now.

**Review by hzxa21 — APPROVED** [review]

**Comment by hzxa21** [comment]
Also, this is actually a user-facing change. Can you update the PR description to include the release note?

File: risedev.yml [hunk 2]
```diff
@@ -20,14 +20,14 @@ profile:
...
       # If you want to use aws-s3, configure AK and SK in env var and enable the following lines:
       # - use: aws-s3
       #   bucket: test-bucket
 
       # If you want to create CDC source table, uncomment the following line
-      # - use: connector-node
+      - use: connector-node
 
       # if you want to enable etcd backend, uncomment the following lines.
       # - use: etcd
```
Comment (BugenZhao) [inline risedev.yml:33]: Please revert these changes.
